app/javascript/react/components/Main.js

```diff
@@ -0,0 +1,16 @@
+import App from './app'
+import React from 'react';
+import { Router, browserHistory, Route, IndexRoute } from 'react-router';
+import DonutIndex from '../containers/DonutIndex'
+
+
+
+const Main = (props) => {
+  return(
+    <Router history={browserHistory}>
+      <Route path='/donuts' component={DonutIndex}/>
+    </Router>
+  )
+}
+
+export default Main;
```

Comment:
ummmm...is this meant to replace your routes in your app.j? You probably didnt mean to make this I am guessing?